hack/verify-godep-licenses.sh

```diff
@@ -18,7 +18,7 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
-KUBE_ROOT=$(dirname "${BASH_SOURCE}")/..
+KUBE_ROOT=$(dirname "${BASH_SOURCE[0]}")/..
```

Comment:
Why does only one file need to export KUBE_ROOT?

Reply:
we probably don't need to export it at all.